problems/arraysIntro.js

```diff
@@ -64,29 +93,51 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
-
+function largerArray(array1, array2) {
+  if(array1.length > array2.length){
+    return array1;
+  } else if (array1.length === array2.length){
+    return "They are the same size"
+  }
+  else {
+    return array2;
+  }
+  }
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
  */
-function middleElement() {}
+function middleElement(array) {
+  let midInx = Math.floor(array.length / 2)
+  if(array.length % 2 === 1){
+    return array[midInx];
+  }else if (array.length % 2 === 0){
+    return [array[midInx - 1], array[midInx]]
+  }
+}
 
 /**
  * Takes in an array and returns the last element without altering or mutating the array
  * @param {Array} array
  * @returns {*}
  */
-function lastElementIndexing() {}
+function lastElementIndexing(array) {
+  let arra1 = array.slice(length - 1)
```

Comment:
The instructions say without altering or mutating the array. slice mutates the array.

We want to key into the last index and return that